src/components/Table/TableBody/TableBodyRow/TableBodyRow.jsx

```diff
@@ -322,17 +322,17 @@ const StyledExpansionTableRow = styled(({ hasRowSelection, ...props }) => <Table
 const StyledTableCellRow = styled(TableCell)`
   &&& {
     ${props => {
-      const { width, offset } = props;
+      const { width, align } = props;
       return width !== undefined
         ? `
         min-width: ${width};
         max-width: ${width};
         white-space: nowrap;
         overflow-x: hidden;
         text-overflow: ellipsis;
-        padding-right: ${offset}px;
+        text-align: ${align};
       `
-        : `padding-right: ${props.offset}px`;
+        : `text-align:${props.align}`;
```

Comment:
should be able to use `align` instead of props.align

Reply:
Code is updated based on your review comments, Please review.